README.md

```diff
@@ -1 +1,165 @@
+    "startDate": "2023-04-24T21:08:35.434000+05:30"
+}
+```
+
+You can use the Execution ARN to see the output of the State Machine execution:
+
+```sh
+awslocal stepfunctions describe-execution \
+    --execution-arn=<EXECUTION_ARN> \
+    --query="output" | jq -r  '. | fromjson'
+```
+
+Replace the `<EXECUTION_ARN>` with the `executionArn` output from the previous step. The result will contain the output of the State Machine execution:
+
+```json
+{
+  "SSN": "123-45-6789",
+  "Amount": 500000,
+  "Term": 30,
+  "Credit": {
+    "Score": 861,
+    "History": 15
+  },
+  "Banks": {
+    "BankAddress": [
+      "BankRecipientPremium",
+      "BankRecipientUniversal",
+      "BankRecipientPawnshop"
+    ]
+  },
+  "Quotes": [
+    {
+      "Quote": {
+        "rate": 3.881919225968371,
+        "bankId": "Premium"
+      }
+    },
+    {
+      "Quote": {
+        "rate": 5.032719931679686,
+        "bankId": "Universal"
+      }
+    },
+    {
+      "Quote": {
+        "rate": 6.019381689166033,
+        "bankId": "PawnShop"
+      }
+    }
+  ]
+}
+```
+
+## GitHub Action
+
+This application sample hosts an example GitHub Action workflow that starts up LocalStack, builds the Lambda functions, and deploys the infrastructure on the runner. You can find the workflow in the `.github/workflows/main.yml` file. To run the workflow, you can fork this repository and push a commit to the `main` branch.
+
+Users can adapt this example workflow to run in their own CI environment. LocalStack supports various CI environments, including GitHub Actions, CircleCI, Jenkins, Travis CI, and more. You can find more information about the CI integration in the [LocalStack documentation](https://docs.localstack.cloud/user-guide/ci/).
+
+## Cloud Pods
+
+[Cloud Pods](https://docs.localstack.cloud/user-guide/tools/cloud-pods/) are a mechanism that allows you to take a snapshot of the state in your current LocalStack instance, persist it to a storage backend, and easily share it with your team members.
+
+You can convert your current AWS infrastructure state to a Cloud Pod using the `localstack` CLI. Check out our [Getting Started guide](https://docs.localstack.cloud/user-guide/tools/cloud-pods/getting-started/) and [LocalStack Cloud Pods CLI reference](https://docs.localstack.cloud/user-guide/tools/cloud-pods/pods-cli/) to learn more about Cloud Pods and how to use them.
+
+To inject a Cloud Pod that contains the infrastructure for this application, you can run the following command:
+
+```sh
+localstack pod load file://$(pwd)/cloud-pod/loan-broker-application
+```
+
+Alternatively, you can use [Cloud Pods Launchpad](https://docs.localstack.cloud/user-guide/tools/cloud-pods/launchpad/) to quickly inject Cloud Pods into your running LocalStack container. Click on the [badge] to launch Cloud Pods Launchpad and inject the Cloud Pod for this application by clicking the `Inject` button.
```

Comment:
Do you plan to add the cloudpod badge here already, or should we remove this section for now?

Reply:
I plan to do that once we merge this.